itl/command-plugins-manubulon.conf

```diff
@@ -292,6 +296,7 @@ object CheckCommand "snmp-interface" {
 	vars.snmp_interface_kbits = true
 	vars.snmp_interface_megabytes = true
 	vars.snmp_interface_64bit = false
+        vars.snmp_interface_v2 = false
```

Comment:
Indent seems wrong here. 

Can you also update the documentation table please?

Reply:
Which file contains the documentation table for these commands?

Reply:
https://github.com/Icinga/icinga2/blob/master/doc/10-icinga-template-library.md